var/spack/repos/builtin/packages/verilator/package.py

```diff
@@ -55,6 +55,7 @@ class Verilator(AutotoolsPackage):
     url      = "https://www.veripool.org/ftp/verilator-3.920.tgz"
 
     version('3.920', '71de7b9ddb27a72e96ed2a04e5ccf933')
+    version('3.904', '7d4dc8e61d5e0e564c3016a06f0b9d07')
 
     depends_on('bison')
     depends_on('flex')
```

Comment:
Would you mind checking these dep types? I think `bison` and `flex` should be `type='build'` and `perl` should be `type=('build', 'run')`